lib/tutil/fixtures.go

```diff
@@ -0,0 +1,21 @@
+package tutil
+
+import (
+	"crypto/ecdsa"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/stretchr/testify/require"
+)
+
+// PrivateKeyFixture generates an ethereum ecds private key.
```

Comment:
nit: `PrivateKeyFixture returns a new random ethereum ecdsa private key and associated address`